### DISCUSSION TITLE
Segmentation fault: address not mapped to object at address

### DISCUSSION THREAD

**Discussion by liuzg-23** [scivision/mumps-superbuild#28] [General]
Hello,
I solved the real system of linear equations using mumps solver on a computer clusters. When I solved a small-scale (1000,000 unknowns), it worked. However, when I solved a large-scale (9000,000 unknowns), the code reported an error:
.....
Calling MUMPS direct solver
Step 3: Solving primal problem...
[cdr2459:238884:0:238884] Caught signal 11 (Segmentation fault: address not mapped to object at address 0x11cc8e00c)
[cdr2344:54672:0:54672] Caught signal 11 (Segmentation fault: address not mapped to object at address 0x11dbe5c6c)
[cdr2404:39251:0:39251] Caught signal 11 (Segmentation fault: address not mapped to object at address 0x11e56fc4c)
......
==== backtrace (tid:  54674) ====
 0 0x000000000001f093 ucs_debug_print_backtrace()  /tmp/ebuser/avx2/UCX/1.9.0/GCCcore-10.2.0/ucx-1.9.0/src/ucs/debug/debug.c:656
 1 0x00000000000130f0 __funlockfile()  :0
 2 0x00000000011fd34f dmumps_init_root_ana_()  ???:0
 3 0x00000000012868ba dmumps_ana_driver_()  ???:0
 4 0x00000000011ef6ee dmumps_()  ???:0
 5 0x0000000001125eb6 dmumps_f77_()  ???:0
 6 0x0000000001123aef dmumps_c()  ???:0
 7 0x00000000004b03ec mfem::MUMPSSolver::SetOperator()  ???:0
 8 0x000000000042c039 GOAFEM::solve()  ???:0
 9 0x0000000000419419 main()  ???:0
10 0x0000000000023e1b __libc_start_main()  /cvmfs/soft.computecanada.ca/gentoo/2020/usr/src/debug/sys-libs/glibc-2.30-r8/glibc-2.30/csu/../csu/libc-start.c:308
11 0x0000000000411c6a _start()  ???:0
=================================
...

Can you help me solve these problems? Thanks very much.

Best wishes, 
Zhengguang.

**Comment by scivision** [comment]
Hi, I think this is a question for the MUMPS Users List: https://listes.ens-lyon.fr/sympa/subscribe/mumps-users
This repository (scivision/mumps) is just to allow building MUMPS via CMake, I don't know about the internals of MUMPS.
